### DISCUSSION TITLE
Why billing address is null when create draft order via API?

### DISCUSSION THREAD

**Discussion by eric-makro** [medusajs/medusa#2407] [Q&A] [answered]
I already put value in billing address during create draft order, but why did billing address become null in response?
![image](https://user-images.githubusercontent.com/110385557/194998541-8d4b1cb9-ca84-49f3-a218-06e2f9c77883.png)
can someone help please?

**Comment by srindom** [comment, marked as answer]
This is a bug - will make sure to open an issue on this 👍

**Reply by eric-makro** [reply]
Thanks @srindom

**Reply by srindom** [reply]
A fix has been merged to develop so will be part of next release 
https://github.com/medusajs/medusa/pull/1902
